index.html

```diff
@@ -39,6 +39,71 @@ <h2>I'am a Software Developer</h2>
 </div>
 </div>
 </div>
+<div class="grid-container">
+    <div class="grid-item1">
+<ul>
+ <li class="head">My Recent Works</li>
+ <li class="border"></li>   
+<li id="img7"></li>
+<li class="post">Multi-Post Stories</li>
+<li class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</li>
+</ul>
+<ul class="tag">
+<li id="tag1">css</li>
+<li id="tag2">html</li>
+  <li id="tag3">bootstrap</li>
+  <li id="tag4">Ruby</li>
+  <li><button class="btn" type="button">See Project</button></li>
+</ul>
+<div>
+  <ul>
+    <li class="img8"></li>
+    <li class="post2">Profesional Art Printing Data</li>
+<li class="support2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</li>
+</ul>
+<ul class="tagB">
+<li id="tag5">html</li>
+  <li id="tag6">bootstrap</li>
+  <li id="tag7">Ruby</li>
+  <li><button class="btn1" type="button">See Project</button></li>
+</ul>
+</div>
```

Comment:
- [ ] To match the original design better, once you are done with the changes mentioned above, you should consider adding at least 2 copies of this section.